ci/taskcluster/bin/decision.py

```diff
@@ -7,15 +7,23 @@
 
 tasks = [
     {
+        'name': 'recipe-client-addon:build',
+        'description': 'Build Firefox with recipe-client-addon',
+        'command': 'normandy/recipe-client-addon/bin/tc/build.sh',
+    },
+    {
         'name': 'recipe-client-addon:test',
         'description': 'Test recipe-client-addon with gecko-dev',
         'command': 'normandy/recipe-client-addon/bin/tc/test.sh',
+        'dependencies': ['recipe-client-addon:build'],
+        'artifacts_from': {
```

Comment:
The `_from` at the end is kinda confusing for a mapping; I couldn't figure out what the key of the mapping was at first. Maybe name this `artifacts`, or use a list with `envvar` keys instead? This is a nitpick though.